addon/components/frost-bunsen-cell/component.js

```diff
@@ -1,12 +1,11 @@
 import _ from 'lodash'
-import Ember from 'ember'
 import computed, {readOnly} from 'ember-computed-decorators'
 
-import PropTypesMixin, {PropTypes} from 'ember-frost-bunsen/mixins/prop-types'
+import FrostComponent, {PropTypes} from 'ember-frost-component'
```

Comment:
@psbanka: Mixin vs. Component... does either win the cage match for you?  Any particular merits to one approach vs the other?  I'm thinking things like, "well, if it's a mixin, it's clear that's totally optional, you are still extending Ember.component, and that is better because..."